src/lib/system-info.ts

```diff
@@ -3,6 +3,10 @@ import * as _ from 'lodash';
 import * as memoizee from 'memoizee';
 import { promises as fs } from 'fs';
 
+// TODO: remove this once we can update Node to v16 and typescript
+// to an es2021 compatible version for native Promise.any
+import { Promise } from 'bluebird';
```

Comment:
`Promise.any` should be available natively, no?

Reply:
Otherwise LGTM!

Reply:
available in node 15 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any#browser_compatibility

Reply:
Ahh, oh no